pkg/nsx/services/store.go

```diff
@@ -78,8 +81,15 @@ func queryGroup(service *SecurityPolicyService, wg *sync.WaitGroup, fatalErrors
 	defer wg.Done()
 	queryParam := fmt.Sprintf("%s:%s", resourceType, resourceTypeGroup) + " AND " + queryTagCondition(service)
 	var cursor *string = nil
+	pageSize := PageSize
 	for {
-		response, err := service.NSXClient.QueryClient.List(queryParam, cursor, nil, nil, nil, nil)
+		response, err := service.NSXClient.QueryClient.List(queryParam, cursor, nil, &pageSize, nil, nil)
+		err = filterError(err)
```

Comment:
I'm not sure if all the query has the same page size limit. if yes, when you find the right page size, then each time you could use it instead of try again.
so PageSize should be updated to the right one.
it seems that you could find the right page size at the first time, then other query no need to try again.

Reply:
@TaoZou1 The max default size is 1000, 60576 error code may appear dynamically, so only have to try again, it is like python code.